agent/engine/docker_task_engine.go

```diff
@@ -397,9 +428,25 @@ func (engine *DockerTaskEngine) ListTasks() ([]*api.Task, error) {
 }
 
 func (engine *DockerTaskEngine) pullContainer(task *api.Task, container *api.Container) DockerContainerMetadata {
-	log.Info("Pulling container", "task", task, "container", container)
+	matches := S3_REGEX.FindStringSubmatch(container.Image)
+
+	if len(matches) != 3 {
+		log.Info("Pulling container", "task", task, "container", container)
```

Comment:
Since that prints out the container specification, I'm tempted to leave it as is.  Plus both upstream methods (`client.PullImage` and `client.ImportImage`) log which image they are importing.